### DISCUSSION TITLE
Provide amd64 build of EarTrumpet

### DISCUSSION THREAD

**Discussion by RavHilton** [File-New-Project/EarTrumpet#726] [Ideas]
I've been using EarTrumpet for a little while now, but as it is now 2021, I'm trying to rid my system of all ancient tech (aka x86 code).  Is there any chance EarTrumpet will release an x64 build (especially for Chocolatey deployment)?

**Comment by riverar** [comment]
Don't think anyone gains with x64 code here, but will transfer this to the Ideas area. Might be something we can add easily for Chocolatey users.
